template/css/dygraph.css

```diff
@@ -47,6 +47,9 @@
 .dygraph-roller {
   position: absolute;
   z-index: 10;
+  border: 1px solid rgb(61, 61, 61);
+  border-radius: 3px;
+  text-align: center;
```

Comment:
Why did we add this roller styling?

Reply:
@rhcarvalho covered it [here](https://github.com/getsentry/sentry-sdk-benchmark/pull/43#discussion_r722212611). Basically Tailwind resets some styles